features/user/managing_administrators/adding_avatar_to_administrator.feature

```diff
@@ -7,7 +7,7 @@ Feature: Adding an avatar to an administrator
     Background:
         Given I am logged in as an administrator
 
-    @ui @api
+    @ui @api @avatar # temporarily enabled to test only the avatar
```

Comment:
In future PRs, I recommend adding a note `TODO` to make it easier to find such spots in the future